db/schema.rb

```diff
@@ -2218,31 +2218,6 @@
             ORDER BY alert_generation_runs.school_id, alert_generation_runs.created_at DESC) latest_runs
     WHERE (enba.alert_generation_run_id = latest_runs.id);
   SQL
-  create_view "electricity_consumption_during_holidays", sql_definition: <<-SQL
```

Comment:
This has moved further down. Don't know if to leave it as is or move it back and re-check in. Have migrated all the way down and back up and still get this....